ts/screens/authentication/NfcStartScreen.tsx

```diff
@@ -0,0 +1,76 @@
+import { Container, H1, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import ScreenHeader from "../../components/ScreenHeader";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../components/ui/FooterWithButtons";
+import I18n from "../../i18n";
+import ROUTES from "../../navigation/routes";
+import variables from "../../theme/variables";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+const styles = StyleSheet.create({
+  contentContainerStyle: {
+    padding: variables.contentPadding
+  },
+  text: {
+    fontSize: variables.fontSizeBase
+  }
+});
+
+class NfcStartScreen extends React.Component<Props> {
+  public render(): React.ReactNode {
+    return (
+      <Container>
+        <BaseScreenComponent goBack={true}>
+          <ScreenHeader
+            heading={<H1>{I18n.t("authentication.cie.enableNfcTitle")}</H1>}
+            icon={require("../../../img/icons/nfc-icon.png")}
+          />
+          <View style={styles.contentContainerStyle}>
+            <Text style={styles.text}>
+              {I18n.t("authentication.cie.enableNfcContent")}
+            </Text>
+            <View spacer={true} />
+            <Text
+              link={true}
+              onPress={(): boolean =>
+                this.props.navigation.navigate(
+                  ROUTES.AUTHENTICATION_SPID_INFORMATION
+                )
+              }
+            >
+              {I18n.t("authentication.cie.enableNfcHelp")}
+            </Text>
+          </View>
+        </BaseScreenComponent>
+        <FooterWithButtons
+          type="TwoButtonsInlineThird"
+          leftButton={{
+            cancel: true,
+            onPress: (): void => {
```

Comment:
`onPress: this.props.navigate.goBack` should work also here. It avoids we create a new instance of the function each time the onPress is called (happening when arrow functions are used)